vm/vmExprFunction.go

```diff
@@ -148,7 +148,7 @@ func callExpr(callExpr *ast.CallExpr, env *Env) (rv reflect.Value, err error) {
 		for i, expr := range callExpr.SubExprs {
 			if addrExpr, ok := expr.(*ast.AddrExpr); ok {
 				if identExpr, ok := addrExpr.Expr.(*ast.IdentExpr); ok {
-					invokeLetExpr(identExpr, args[i].Elem().Elem(), env)
+					invokeLetExpr(identExpr, args[i].Elem(), env)
```

Comment:
@mattn 
So this reverts the previous commit you did. May I ask what you were trying to fix? Can I add a test for what you were trying to fix?

I added a test that will cause a panic if this is not change.

Reply:
first Elem() is for dereference the pointer-variable. second one is for dereference the value that was set via v.Set (should be reflect.Interface).

Reply:
But it might not be a interface, right?

Reply:
sure.